### PR TITLE
Fix cmake resources

### PR DESCRIPTION
**Pull request by dlyr**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Mostly bug fix + a small feature

* **What is the current behavior?** (You can also link to an open issue here)
there is a type in cmake function BUILDLOCATION instead of BUILD_LOCATION
also PREFIX arg provided by installTargetResources is not forwarded from configure_radium_app function
lastly application installed out of radium dir have radium resources installed in bin/resrouces, instead of bin/../resources

* **What is the new behavior (if this is a feature change)?**
Now app resources can be installed in a specific path using prefix
Radium lib resources are installed in the Resources dir instead of bin/Resources, this is more consistent.
May need some doc update.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
nope

* **Other information**:

ready to review, tested with my own app on linux, need windows and macos test, and doc.
we may discuss how to automate prefix. Without prefix, app resources and radium resources can have name clash or even use Radium prefix for radium lib resources.

### CODE REVIEW

**Comment by MathiasPaulin** [comment]
> ready to review, tested with my own app on linux, need windows and macos test, and doc.
> we may discuss how to automate prefix. Without prefix, app resources and radium resources can have name clash or even use Radium prefix for radium lib resources.

I'll test it on MacOs and Windows. I'll also add windows installation support for Plugins that, for now, are not usable on windows.
Until all the test and Plugins installation procedure are operational on Windows, do not merge this PR.

**Review by MathiasPaulin — DISMISSED** [review]

File: cmake/radium_setup_functions.cmake
Comment (dlyr) [inline cmake/radium_setup_functions.cmake:58]: before merging, i'm for cleaning the naming convention here:
_ names, without up case.

**Review by MathiasPaulin — DISMISSED** [review]
Tested on macOS and windows, using several Radium-based libraries, plugins and applications. Seems fine. ready to squash and merge.

**Review by MathiasPaulin — APPROVED** [review]

**Pull request merged**
